services/web/server/src/simcore_service_webserver/clusters/handlers.py

```diff
@@ -20,77 +26,103 @@
 
 logger = logging.getLogger(__name__)
 
-# API ROUTES HANDLERS ---------------------------------------------------------
+
+def _handle_cluster_exceptions(handler: Handler):
+    # maps API exceptions to HTTP errors
+    @functools.wraps(handler)
+    async def wrapper(request: web.Request) -> web.Response:
+        try:
+            return await handler(request)
+
+        except ClusterPingError as exc:
+            raise web.HTTPUnprocessableEntity(reason=f"{exc}") from exc
```

Comment:
I think you are missing the ValidationError here or is it auto-handled in the middlewares?

Reply:
All the request inputs (i.e. path/query parameters + body) are validated using the ``servicelib.aiohttp.requests_validation`` and the ValidationERrors handled into http errors in there as well . What other ValidationError might be raise?

Reply:
I think that is all ok then